diff --git a/CMakeLists.txt b/CMakeLists.txt
index a3f1d19c..b56e72d0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,7 @@ endif()
 #--------------------------------------
 # Find libdl
 ign_find_package(
-  DL REQUIRED PRIVATE PRETTY libdl PURPOSE "Required for plugins")
+  DL PRIVATE PRETTY libdl PURPOSE "Required for plugins")

 #--------------------------------------
 # Find dependencies that we ignore for Visual Studio
